docs: add configure connector (en-US) (#97)
logto-io/docs: docs/docs/recipes/configure-connectors/social-connector.md

---
sidebar_label: Social Connector
sidebar_position: 2
---

# Social Connector

Suppose you are now on the "Connectors" page in Admin Console. Go to the "Social connectors" tab and add a new connector to enter the configuration process.

:::tip

> You may find that no secondary choice of **_platform_** is required for some connectors such as Google, Facebook, and GitHub. These connectors' **_platform_** are always **_Universal_**. Explore **_platform_** description (TODO: add a link here after connector reference merged) to know more.

Reviewer: generally we don't need italic and bold at the same time.
```suggestion
You may find that some connectors do not have a secondary choice of platform, such as Google, Facebook, and GitHub. These connectors are _Universal connectors_. Explore platform description (TODO: add a link here after connector reference merged) to know more.
```

Choose a proper **_platform_** from either **_Web_** or **_Native_** for connectors to fit your use case.<br/>
You can set up a single **_Native_** connector w/o adding a **_Web_** connector if you only provide mobile apps and vice versa.
:::

> Follow the README to compose the connector config JSON with little effort.

Reviewer: ditto
